app/backend/src/tests/test_groups.py

```diff
@@ -262,8 +213,91 @@ def test_ListOtherUserGroups(testing_communities):
             assert [g.group_id for g in res.groups] == [hitchhikers_id, foodies_id, skaters_id]
 
 
+# These tests are separate from above as they mutate the database
```

Comment:
`testing_communities` takes quite a while to set up. Would it be better to add some hidden users in that setup script and then not mutate the database with `make_user_invisible` within the test at all?

Reply:
If it's faster, then yes, that would make more sense

Reply:
Yeah, that's the reason for the somewhat unusual class setup in this test file. Thanks!